spec/support/pages/bookable_slots.rb

```diff
@@ -0,0 +1,20 @@
+module Pages
+  class BookableSlotDay < SitePrism::Section
+    element :appointment_count, '.t-appointment-count'
+  end
+
+  class BookableSlots < SitePrism::Page
+    set_url '/locations/{location_id}/bookable_slots'
+
+    def day(date_or_datetime, am_or_pm)
```

Comment:
As I mentioned in slack - perhaps we can use the ID from the earlier created slot to find these elements instead?